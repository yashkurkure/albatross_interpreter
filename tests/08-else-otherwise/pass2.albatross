while (a <> 10) {
  a := a + 1;
} otherwise {
  return 3;
}
