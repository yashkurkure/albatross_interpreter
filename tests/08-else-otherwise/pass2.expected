((a)<>(10))
((a)+(1))
(3)

